src/utils/eis_utils/src/eis_coms.c

```diff
@@ -152,7 +153,7 @@ typedef struct tagEIS_HTTP_WORKLOAD_CONTEXT
 /**
  * @brief Maximum amount of bytes for any EIS response
  */
-#define EIS_RESP_SIZE_MAX 4096
+#define EIS_RESP_SIZE_MAX 8192
```

Comment:
Is this the true max, or should x2 for safety factor?

Reply:
x2 for safety factor at the moment.